rae_bringup/scripts/led_test.py

```diff
@@ -3,12 +3,11 @@
 from rclpy.node import Node
 from rclpy.time import Time
 
-from std_msgs.msg import ColorRGBA
+from rae_msgs.msg import ColorPeriod
```

Comment:
This is for testing purposes?

Reply:
No - I created a new message so we can control individual LEDs - I can rename message type, I didnt really put any thought into that part. Besides color each led will get value that determines its frequency for intensity. Higer frequencies will make LED blink, lesser ones will make it slowly pulse. That way we can easily control separate LEDs differently (example in car app I need leds on the top right/left side of the strip to blink, but I need front ones to remain at constant intensity).